### PR TITLE
Microverse reviewer Role

### PR DESCRIPTION
**Pull request by gids-dadi**
The pull request is on the branch "Microverse reviewer role" meant to provide platform to practice code reviewing.

### CODE REVIEW

**Review by thecodechaser — CHANGES_REQUESTED** [review]
### Hi @gids-dadi 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is invalid because you did not perform code review, Please perform code review and submit again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
-# Microverse-review-project
\ No newline at end of file
+#Microverse-code review project
+
+# Description of the project.
+
```
Comment (gids-dadi) [inline README.md:4]: Great description in the README file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        </ul>
+      </div>
+      <a
+        href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+        rel="noopener"
+        id="tribute-link"
+        target="_blank"
+        >Read more</a
+      >
+    </main>
```
Comment (gids-dadi) [inline index.html:68]: Good use of semantics.

File: .github/workflows/linters.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (gids-dadi) [inline .github/workflows/linters.yml:1]: Great! No linter error.

**Review by gids-dadi — COMMENTED** [review]
Great work! 
There are no other changes required for this project, you can proceed to your next project.

**Pull request closed without merging**